0: Traceback (most recent call last):
0:   File "/ccc/scratch/cont003/mds/guerouda/DASK_IPCA_WCONF_SAME/E3/P32_W16_D256_D2023-01-24_11:38:59/ipca_post.py", line 61, in <module>
0:     print("explained_variance ,singular_values",explained_variance.result() ," and " , singular_values.result(), flush=True )
0:   File "/ccc/work/cont003/gen2224/gen2224/spack/var/spack/environments/gysela-deisa-irene-skl-draft/.spack-env/view/lib/python3.9/site-packages/distributed/client.py", line 229, in result
0:     raise exc.with_traceback(tb)
0: KeyError: 'array-original-9d8bfeb3fedcdf73faab011f0f157a0f'
srun: error: irene2125: task 0: Exited with exit code 1
